src/Runtime.php

```diff
@@ -26,6 +27,8 @@ class Runtime
     private ImagineInterface $imagine;
     private bool $useLiipDefaultImage;
     private ?string $liipDefaultImage;
+    private string $env;
+    private FallbackImageResolver $FallbackImageResolver;
```

Comment:
No uppercase on first letter for a variable name

Reply:
Fixed (wrong search and replace)